src/signals/incident-management/containers/IncidentDetail/components/StatusForm/components/DefaultTexts/DefaultTexts.tsx

```diff
@@ -29,7 +29,11 @@ const DefaultTexts: FC<DefaulTextsProps> = ({
     defaultTexts.find((text) => text.state === status)
 
   return (
-    <ModalDialog title="Standaardtekst" onClose={onClose}>
+    <ModalDialog
+      title="Standaardtekst"
+      onClose={onClose}
+      isConfirmation={false}
```

Comment:
default is `false`, so you don't have to specify that here.

Reply:
Removed it.